feature-libs/user/account/components/login-form/login-form.module.ts

```diff
@@ -58,6 +58,17 @@ import { LoginFormComponent } from './login-form.component';
             },
           ],
         },
+        CustomLoginComponent: {
+          component: LoginFormComponent,
+          guards: [NotAuthGuard],
+          providers: [
+            {
+              provide: LoginFormComponentService,
+              useClass: LoginFormComponentService,
+              deps: [AuthService, GlobalMessageService, WindowRef],
+            },
+          ],
+        },
```

Comment:
`query:` could you help me understand why we are adding the separate configuration, when we agreed on reusing existing CMS component `ReturningCustomerLoginComponent`? I might be missing something obvious.

Reply:
I forgot to remove it, sorry